app/lib/imports/missing_move_ending_events.rb

```diff
@@ -32,12 +32,12 @@ def results
     @results ||= records.each_with_object(Imports::Results.new) do |record, results|
       move = Move.find_by(id: record[:move_id], status: EXPECTED_MOVE_STATUSES)
       if move.nil?
-        results.record_failure(record)
+        results.record_failure(record, reason: 'Could not find move.')
         next
       end
 
       unless ALLOWED_EVENT_TYPES.include?(record[:event_type])
-        results.record_failure(record)
+        results.record_failure(record, reason: 'Event type not allowed.')
```

Comment:
Minor - might be me but couldn't see a test case for this.

Reply:
Good point, I've amended the commit to include that now.